authors/apps/articles/views.py

```diff
@@ -194,3 +196,49 @@ def get(self, request, slug):
         error_message = 'Sorry article with this slug doesnot exist'
         return Response({'errors': error_message},
                         status=status.HTTP_404_NOT_FOUND)
+class ArticleRatingAPIView(generics.GenericAPIView):
+    """
+    Allows user to rate an article.
+    Restricts author from rating their own article
+    """
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = serializers.ArticleRatingSerializer
+    renderer_classes = (ArticleJSONRenderer,)
+
+    def post(self, request, slug):
+        """
+        Allow user to rate an article
+        :param request: The incoming request object
+        :param slug: The expected article slug
+        :return: article's rate score, title, author of that rated article
+        """
+
+        article = model_helpers.get_single_article_using_slug(slug)
+        user = request.user
+
+        rate = request.data.get('rate_score', {})
+        if rate > 5 or rate < 1:
+            return Response({'errors': 'Rate score should be a value > 1 and < 5',
```

Comment:
the number of characters too many for a single line, should not exceed 79. `PEP8`